subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -318,15 +317,9 @@ private void fillInFailureResolution(FailureDetails details) {
             );
         }
 
-        boolean hasCompileError = hasNonGradleSpecificCauseInAncestry &&
-            hasCauseAncestry(details.failure, CompilationFailedIndicator.class);
         LogLevel logLevel = loggingConfiguration.getLogLevel();
         boolean isLessThanInfo = logLevel.ordinal() > INFO.ordinal();
-        if (hasCompileError && isLessThanInfo) {
-            context.appendResolution(output ->
-                runWithOption(output, INFO_LONG_OPTION, " option to get more log output.")
-            );
-        } else if (logLevel != DEBUG && !hasNonGradleSpecificCauseInAncestry) {
```

Comment:
ℹ️ as the compilation error will provide these messages from now on, we can simplify this code path